examples/RDF_to_LPG_Example.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
This file isn't in the index.ipynb which links all the examples. Lets add a section to the index called `Workflow Examples` and add this notebook with the label `Collapsing RDF to a Latent Property Graph (networkx)`.